exec/plugin-commands-script-runners/src/dlx.ts

```diff
@@ -174,10 +176,23 @@ async function getPkgName (pkgDir: string): Promise<string> {
   return dependencyNames[0]
 }
 
-async function getBinName (modulesDir: string, pkgName: string): Promise<string> {
+async function getBinName (modulesDir: string, pkgName: string, engineStrict: boolean): Promise<string> {
   const pkgDir = path.join(modulesDir, pkgName)
   const manifest = await readPackageJsonFromDir(pkgDir)
   const bins = await getBinsFromPackageManifest(manifest, pkgDir)
+  const warn = checkPackage(pkgName, manifest, {
+    nodeVersion: process.version,
```

Comment:
Do not pass in the node version. This is not the correct node version, when pnpm is bundled with node.js. checkPackage checks the correct node version, no need to pass in an option.